spec/java_buildpack/component/base_component_spec.rb

```diff
@@ -51,6 +53,13 @@
     expect(stdout.string).to match(/Downloading Stub Base Component #{version} from #{uri}/)
   end
 
+  it 'downloads and installs InstallAnywhere (tm) BIN file in the sandbox',
```

Comment:
This functionality is IBM JRE specific and therefore shouldn't be tested as part of this class.